watertap/unit_models/zero_order/ozone_aop_zo.py

```diff
@@ -29,31 +29,58 @@ class OzoneAOPZOData(OzoneZOData):
     Zero-Order model for a Ozone-AOP unit operation.
     """
 
-    CONFIG = OzoneZOData.CONFIG()
-
     def build(self):
         super().build()
 
         self._tech_type = "ozone_aop"
 
-
         self.oxidant_dose = Var(self.flowsheet().time,
```

Comment:
In other models, I think we have kept the dosage in mg/L and just done unit conversions as necessary.